ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -6,37 +6,37 @@ set -o pipefail
 
 echo "************ baremetalds packet setup command ************"
 
-# Initial check
-if [ "${CLUSTER_TYPE}" != "packet" ] ; then
-    echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
-    exit 1
-fi
-
-ansible_home=/tmp/ansible
-mkdir -p ${ansible_home}
-
-export ANSIBLE_LOCAL_TEMP=${ansible_home}
-
 # Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
 # to be able to SSH.
 if ! whoami &> /dev/null; then
     if [[ -w /etc/passwd ]]; then
-        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${ansible_home}:/sbin/nologin" >> /etc/passwd
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
     else
         echo "/etc/passwd is not writeable, and user matching this uid is not found."
         exit 1
     fi
 fi
 
-# Prepare Ansible playbook
-cat > ${ansible_home}/setup.yaml <<-EOF
-- name: creating Packet host {{ packet_hostname }}
+# Prepare Ansible playbook and environment
+ansible_home=/tmp/ansible
```

Comment:
nit: let's use all capitals for bash variables primarily for consistency, and also to better distinguish them from ansible native variables